params/superchain.go

```diff
@@ -75,46 +75,32 @@ func LoadOPStackChainConfig(chainID uint64) (*ChainConfig, error) {
 		TerminalTotalDifficultyPassed: true,
 		Ethash:                        nil,
 		Clique:                        nil,
-		Optimism: &OptimismConfig{
-			EIP1559Elasticity:        6,
-			EIP1559Denominator:       50,
-			EIP1559DenominatorCanyon: newUint64(250),
-		},
+		Optimism:                      nil, // Set below
```

Comment:
we could just remove all the `nil`-setting...

Reply:
I ended up refactoring this here [2d89785](https://github.com/ethereum-optimism/op-geth/pull/371/commits/2d89785aecb66432feac1e1016d4753847a67abb) so we never set it to `nil` in the end.